compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BIntersectionType.java

```diff
@@ -94,6 +95,11 @@ public boolean isNullable() {
 
     @Override
     public String toString() {
+        Name name = this.tsymbol.name;
+        if (((this.tsymbol.flags & Flags.ANONYMOUS) != Flags.ANONYMOUS) && !name.value.isEmpty()) {
```

Comment:
Shall we use org.wso2.ballerinalang.compiler.semantics.model.symbols.Symbols#isFlagOn instead?